### ISSUE TITLE
please explain networking for media docker-compose

### ISSUE THREAD

**Issue by BlinkyStitt** [TimJDFletcher/dockerfiles#15]
Hello! I just came across your containers for samba+time machine and it is exactly what I needed. So thank you for that! Then I started browsing this repo and I saw your docker-compose file for routing some services through Tor. Can you explain any extra setup steps needed to make this work? 

I'm guessing the torproxy container works out of the box, and that you just have to set anything inside the `internal` network to use `tor:8118` as a web proxy or `tor:9050` as a socks proxy. But wanted to be sure I wasn't missing anything.

https://github.com/TimJDFletcher/dockerfiles/blob/0101aedd633f5cbde5a40649e34d2176e1201a06/media/docker-compose.yml#L19-L28

**Comment by TimJDFletcher** [comment]
You are correct that is the design, and I restricted the internal network to prevent any traffic escaping that was not sent via tor
